pkg/sql/logictest/testdata/logic_test/zone_config

```diff
@@ -313,6 +317,9 @@ skipif config local-legacy-schema-changer
 statement error pq: columns is a virtual object and cannot be modified
 ALTER TABLE information_schema.columns CONFIGURE ZONE USING gc.ttlseconds = 100000
 
+statement ok
+SET CLUSTER SETTING sql.schema.force_declarative_statements = "!CONFIGURE ZONE"
```

Comment:
this might be safer with a `RESET CLUSTER SETTING` instead. it should be one less line we have to remember to delete later then.